### ISSUE TITLE
resnet50+acb

### ISSUE THREAD

**Issue by anti199994** [DingXiaoH/ACNet#15]
The author, I used the Acb module in resnet50, and the training improvement was super slow. Only 10 epochs increased by 2 percentage points, and many epochs were increased. Moreover, the distance to the target was still 20 percentage points. I wonder if there was any fitting @DingXiaoH

**Comment by DingXiaoH** [comment]
If it does not work on your specific model and dataset, based on my experience, I would suggest you
1. try different learning rate schedules
2. initialize the trained scaling factor of batch norm (e.g., gamma variable in Tensorflow and bn.weight in PyTorch) in the three branches of every ACB as 1/3.
